scripts/cloudfront_routing.js

```
@@ -0,0 +1,41 @@
+/**
+ * This script follows ES5.1 rules for compatibility.
+ * Avoid ES6+ syntax (e.g., `let`, `const`, arrow functions).
+ */
+'use strict';
+function handler(event) {
```

ESLint

[error] 'handler' is defined but never used.

(no-unused-vars)